### ISSUE TITLE
Parsing HTML comments.

### ISSUE THREAD

**Issue by fisx** [bgamari/html-parse#1]
```haskell
Text.HTML.Parser> tagStream "<p><!-- wef --></p>"
[TagOpen "p" [],ContentChar '<',ContentText "!-- wef -->",TagClose "p"]
```

I would have expected:

```haskell
[TagOpen "p" [],Comment "wef",TagClose "p"]
```

**Comment by bgamari** [comment]
Oh dear; how embarrassing!

This is fixed by a34cdf77980ed6d720dbcd283e4328367d337fb1.

**Comment by fisx** [comment]
(-:  this is why testing is awesome!  thanks for the quick fix!
